apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,37 @@
 'use strict';
-
-$(document).ready(function() {
+var $block = null;
+$(document).ready(function () {
   // Put app logic here
+    $('[data-stack]').click(function () {
+      //picks up the block
+      if (!$block){
```

Comment:
This if/else reads like this:
"IF NOT block, then pick it up, ELSE put it down"
The double negative in the else case is a little confusing, IMO

I'd prefer to see:
"IF block, put it down, ELSE pick it up"

Just a smidge easier to read